test/FunctionMetadataGeneratorTests/FunctionMetadataGeneratorTests.cs

```diff
@@ -176,7 +176,7 @@ public void StorageFunctions()
             AssertDictionary(extensions, new Dictionary<string, string>
             {
                 { "Microsoft.Azure.WebJobs.Extensions.Storage.Queues", "5.1.3" },
-                { "Microsoft.Azure.WebJobs.Extensions.Storage.Blobs", "5.1.2" },
+                { "Microsoft.Azure.WebJobs.Extensions.Storage.Blobs", "" },
```

Comment:
Is this intentionally an empty string for blob versioning?

Reply:
yikes, definitely not - find and replace gone wrong :)